public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -326,7 +326,7 @@ export const ReceiversTable = ({ config, alertManagerName }: Props) => {
   };
 
   const receivers = useGetReceiversWithGrafanaAppTypes(config.alertmanager_config.receivers ?? []);
-  const rows: RowItemTableProps[] = useMemo(() => {
+  let rows: RowItemTableProps[] = useMemo(() => {
```

Comment:
Do we override this variable anywhere?

Reply:
Nope! This is a left-over of my local testing where I just duplicate items in the list to simulate 10.000 items, will revert :)